scripts/deploy.py

```diff
@@ -171,28 +181,30 @@ def f(ctx):
             click.secho('Git returned non-zero exit code!', fg='red')
             return False
         ctx['cherrypick-commit'] = SelectionPrompt(
-            "Select commit to cherrypick.",
-            [(x, x.split()[0]) for x in commits],
-            zero_choice="[Select 0 to finish.]")
+            'Select commit to cherrypick.', [(x, x.split()[0])
+                                             for x in commits],
+            zero_choice='[Select 0 to finish.]')
         if ctx['cherrypick-commit'] is None:
             raise Jump(jump_on_end)
         return True
 
-    f.__doc__ = "Choose which commit to cherrypick"
+    f.__doc__ = 'Choose which commit to cherrypick'
     return f
 
 
 def CherryPick(ctx):
     """Cherry-pick a commit"""
-    return RunCmdStep('git cherry-pick %s' % ctx['cherrypick-commit'])(ctx)
+    return RunCmdStep(f'git cherry-pick {ctx["cherrypick-commit"]}')(ctx)
 
 
 def WriteVersionToFile(version):
-    with open(VERSION_PATH, "w") as f:
+    with open(VERSION_PATH, 'w') as f:
         f.write('// This file is automatically generated by deploy.py. '
                 'Do not edit this file.\n')
         for k, v in version.items():
-            f.write('#define LC0_VERSION_%s %s\n' % (k, json.dumps(v)))
+            # Value may be string or number. JSON formatting of those data
+            # types is conveniently the same as in C++, so outputting JSON.
+            f.write(f'#define LC0_VERSION_{k} {json.dumps(v)}\n')
```

Comment:
I guess the part that had confused me before is that the string should already be suitably interpreted/converted/whatever when it's placed into the `version` dict. For doing string to string concatenation, there should be no interpretation/converted/whatever.